examples/se2_localization_liekf.cpp

```diff
@@ -0,0 +1,282 @@
+/**
+ * \file se2_localization_liekf.cpp
+ *
+ *  Created on: Jan 20, 2021
+ *     \author: artivis
+ *
+ *  Modified on: Aug 30, 2024
+ *     \author: chris
+ *
+ *  ---------------------------------------------------------
+ *  This file is:
+ *  (c) 2021 artivis
+ *
+ *  This file is part of `manif`, a C++ template-only library
+ *  for Lie theory targeted at estimation for robotics.
+ *  Manif is:
+ *  (c) 2021 artivis
+ *  ---------------------------------------------------------
+ *
+ *  ---------------------------------------------------------
+ *  Demonstration example:
+ *
+ *  2D Robot localization based on position measurements (GPS-like)
+ *  using the (Left) Invariant Extended Kalman Filter method.
+ *
+ *  See se2_localization.cpp for the right invariant equivalent
+ *  ---------------------------------------------------------
+ *
+ *  This demo corresponds to the application in chapter 4.3 (left invariant)
+ *  in the thesis
+ *  'Non-linear state error based extended Kalman filters with applications to navigation' A. Barrau.
+ *
+ *  It is adapted after the sample problem presented in chapter 4.1 (left invariant)
+ *  in the thesis
+ *  'Practical Considerations and Extensions of the Invariant Extended Kalman Filtering Framework' J. Arsenault
+ *
+ *  Finally, it is ported from an example (a matlab code plus a problem
+ *  formulation/explanation document) by Faraaz Ahmed and James Richard
+ *  Forbes, McGill University.
+ *
+ *  The following is an abstract of the content of the paper.
```

Comment:
You have to say which paper, because it is none of the above. The paper is micro Lie theory.